test/js/node/http/node-http.test.ts

```diff
@@ -829,6 +829,90 @@ describe("node:http", () => {
     });
   });
 
+  describe("NODE_EXTRA_CA_CERTS", () => {
+    const run = async (done, env, func: (proc: Subprocess<"pipe">) => Promise<void>) => {
+      const server = https.createServer(
+        {
+          key: nodefs.readFileSync(path.join(import.meta.dir, "fixtures", "openssl_localhost.key")),
```

Comment:
Looks like these files were not committed?

Reply:
Whoops, I initially based this off of #11322. I'll rebase now that that's merged and we can reuse the same certs.

Reply:
@LudvigHz 
Is there any update for the status of this PR?
🙏

Reply:
I haven't had the time to work on it lately. And as I'll need to refactor alot of the http initialization it's going to require quite a bit more work than what this PR proposes currently.

But hopefully I'll have some free time soon to look at it some more :)